showcases/react-showcase/src/components/popover/index.tsx

```diff
@@ -15,19 +16,27 @@ const getPopover = ({
 	content,
 	animation
 }: DBPopoverProps & { content: string }) => (
-	<DBButton describedbyid={id}>
-		{children}
-		<DBPopover
-			width={width}
-			gap={gap}
-			spacing={spacing}
-			placement={placement}
-			animation={animation}
-			delay={delay}
-			id={id}>
-			{content}
-		</DBPopover>
-	</DBButton>
+	<DBPopover
+		slotTrigger={<DBButton>{children}</DBButton>}
+		width={width}
+		gap={gap}
+		spacing={spacing}
+		placement={placement}
+		animation={animation}
+		delay={delay}
+		id={id}>
+		{content ? (
+			content
+		) : (
```

Comment:
```suggestion
		{content ?? (
```

JSX couldn't do this?

Reply:
fixed